velox/common/base/CMakeLists.txt

```diff
@@ -21,4 +21,6 @@ add_library(velox_common_base BitUtil.cpp RawVector.cpp SimdUtil.cpp)
 target_link_libraries(velox_common_base velox_exception velox_process)
 
 add_subdirectory(test_utils)
-add_subdirectory(benchmarks)
+if(${VELOX_ENABLE_TESTING})
+  add_subdirectory(benchmarks)
```

Comment:
In the future, we could probably separate benchmarks from testing via a different option.